### ISSUE TITLE
Update info group for infos on RW-Parking (RB) (Approved on PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#19370]
### The portal consistently displays an info header "Tree Ordinance Review" for parking permits, so we need to update the info group. While we're at it we're going to update the ungrouped infos as well. 

**For the infos below, update the info group to` Main Permit Information `** 
- [x] ~Reason for Request | 75940 ‼️ `enterprise info`~
- [ ] Approval of Owner/Occupant | 75750
- [ ] Rate if metered (in $ hourly meter rate) | 75948
- [x] ~Activate Upon Payment 75001 ‼️ `enterprise info`~
- [ ] Associated with a Special Event Street Closure but outside of the Closure | 83622
- [ ] Within a Special Event Street Closure Permit | 83621

**Comment by TracyLinder** [comment]
@Nadin-Nader Do you think we can get this on deck by 10/21?

**Comment by rajeshlaunch** [comment]
@KyleErich These info codes are tied up with other Folder types. Info code and Info Group are one to one relation in Amanda, So if we change the info group for one permit it will impact all other permit types as well. Please let us know how do you want to proceed?

**Comment by rajeshlaunch** [comment]
DEV deployment plan submitted for this change

**Comment by Nadin-Nader** [comment]
This has been sent to the DevOps CM Team to be deployed to DEV under **SCTASK0469582**

**Comment by Nadin-Nader** [comment]
@rajeshlaunch this has been deployed to DEV Please UAT

**Comment by rajeshlaunch** [comment]
Deployment plan submitted for TEST

**Comment by Nadin-Nader** [comment]
This has been sent to the DevOps CM Team to be deployed to TEST under **SCTASK0472717**

**Comment by Nadin-Nader** [comment]
@rajeshlaunch this has been deployed to TEST and is ready for UAT

**Comment by rajeshlaunch** [comment]
@Nadin-Nader I reviewed the changes in TEST and it is ready for User Testing in TEST.

**Comment by Nadin-Nader** [comment]
@KyleErich This is ready for UAT on TEST

**Comment by KyleErich** [comment]
@Nadin-Nader @rajeshlaunch I'm OK approving on Test, though I just want to be sure one more time -- there's no way to hide the Tree Ordinance Review info header without affecting it globally across all folder types?

**Comment by rajeshlaunch** [comment]
@KyleErich  Yes that is correct, we tried it and that is what the restriction we found unfortunately. I'm preparing a PROD deployment for this.

**Comment by rajeshlaunch** [comment]
Production deployment requested for this change.

**Comment by Nadin-Nader** [comment]
This has been sent tot he DevOps CM Team to be deployed to PROD under**SCTASK0479836**

**Comment by Nadin-Nader** [comment]
@rajeshlaunch @KyleErich  this has been deployed to PROD Please UAT

**Comment by Nadin-Nader** [comment]
@TracyLinder @rajeshlaunch  this has been approved by Russell on PROD, is there anything else missing? 
Here are Russell's notes:

> since I wasn't part of the UAT or scoping I have some questions, 
>  
> Did we abandon removing the strikethrough fields shown in github due to them being tied up with other folder types? 
> ~Reason for Request | 75940 ‼️ enterprise info~
> This field is still shown in PROD in the info tab.
>  ~Activate Upon Payment 75001 ‼️ enterprise info~
> This field is still shown in PROD in the info tab.
>  
> The description indicates these fields to be shown in the portal; however, I don't see any of them in the portal but rather in AMANDA which would align with what Kyle and Raj discuss throughout the comments. 
>  
> The Info Group Main Permit Information has the following fields listed:
> Approval of Owner/Occupant*
> Rate if metered ($)*
> Activate Upon Payment
> Within a Special Event Street Closure Permit*
> Associated with a Special Event Street Closure but outside of the Closure*
> ROW Inspector
> The Info Group Tree Ordinance Review has the following fields listed:
> Reason for Request?*
>  
> ### **So, if I'm seeing what Kyle previously approved in Test then I would say the UAT in Prod is successful.**
> 
> 
>  I will add that Activate Upon Payment is now showing under Main Permit Information on other folder types instead of Ungrouped just to document that what was expected to occur with changing the info groups carrying forward across all folder types
>

**Comment by TracyLinder** [comment]
The AC is delivered so I'm going to close this with a note that we didn't fix the original issue with this info showing as Tree Ordinance on the portal. @Nadin-Nader is going to bring it up at Feb's AOB to see if DSD is ok with changing the info group.

**Comment by Nadin-Nader** [comment]
<img width="720" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0tmQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--df14fc960a923d7b469e6fa68c3f6967acbcfd10/image.png" alt="image.png" />

**Comment by TracyLinder** [comment]
@Nadin-Nader reminder to speak to Deanna about how to ask the enterprise if they care if we update the info group on the info Reason for Request.
